### ISSUE TITLE
Not working in AWS Lambda

### ISSUE THREAD

**Issue by michaelalhilly** [mzabriskie/axios#582]
I am unable to get a response from Auth0. Is there a limitation to using axios in Lambda?

```
"use strict";

import axios from "axios";

var site_update = (event, context, callback) => {

	// Gets HTTP request body.

	var body = JSON.parse(event.body);

	// Authenticates user.

	var match = body.authorizationToken.match(/^Bearer (.*)$/);

	var auth_token = match[1];

	axios({
		method: "post",
		url: "https://villagecity.auth0.com/tokeninfo",
		data: {
			id_token: auth_token
		}
	}).then(response => {

		callback(null, {
			statusCode: 200,
			body: JSON.stringify(response)
		});

	}).catch(error => {

		callback(null, {
			statusCode: 300,
			body: JSON.stringify(error)
		});
	})
}
```

**Comment by mzabriskie** [comment]
I have not worked with Lambda before. Are you getting any error messages?

**Comment by michaelalhilly** [comment]
Hi @mzabriskie. Unfortunately not. I checked the logs but depending on the error Lambda may not output any info which was the case here.

I got this working with a completely different approach so I'm not motivated to check this version of the code. Now that I look at the code I submitted I see a potential issue on my part.

Really sorry!

I'm going to close this issue unless you object or request that I test with Lambda.
